tests/topotests/ospf_metric_propagation/r2/frr.conf

```diff
@@ -8,18 +8,18 @@ interface r2-eth0
  ip address 10.0.1.2/24
  ip ospf cost 100
  ip ospf hello-interval 1
- ip ospf dead-interval 30
+ ip ospf dead-interval 15
```

Comment:
I support the idea of lowering the dead-interval.. I'm not sure that sending 15 hellos in 15 seconds is very normal though.

Reply:
I still got failures with 15! some systems still need more time, setting this back to 30.